### ISSUE TITLE
How to ignore patterns?

### ISSUE THREAD

**Issue by vict0rsch** [KulkarniKaustubh/fzf-dir-navigator#1]
Hey!

Trying to love the tool, but searching from `~` is long ; I'd like to exclude patterns like "any `.git`" directory or "any `.cache`" directory or "any `node_modules`" directory etc. you get it :)

Any idea how to do that?

I tried to put the following in my `~/.zshrc` but it's not helping much, `~/.cache/` is still listed by `fzf-dir-navigator`

```zsh
_fzf_compgen_path() {
  fd --follow --exclude ".git" --exclude "node_modules" --exclude ".venv" --exclude ".rvm" --exclude ".cache" --exclude ".virtualenvs" . "$1"
}
```

Thanks

**Comment by KulkarniKaustubh** [comment]
Hey @vict0rsch,
The latest release should be a convenient way to exclude directories from your search results. Please take a look.

Thanks for the issue, and if it helps solve your problem, do close the issue too! Cheers!

**Comment by vict0rsch** [comment]
That's great thanks! 

Do we have to modify the conf file in `~/.oh-my-zsh/custom/plugins/fzf-dir-navigator`? It would be a bit odd, especially with `git` tracking and pulling future updates

**Comment by KulkarniKaustubh** [comment]
I did not think of that initially. I have now updated instructions in the README file as well as the configuration file. The essential design remains the same (mainly because I am not able to come up with a better idea to handle the configurable options), in that a conf file within the repository itself will have to be edited.

The default config file `fzf-dir-navigator.conf` will receive updates, and what is expected from the user is to copy this default config file and name it `fzf-dir-navigator-custom.conf`, so that any pushes to the default config file will not affect the user (as you mentioned). So, in your case, you will have to do the following:

```
$ cp ~/.oh-my-zsh/custom/plugins/fzf-dir-navigator.conf ~/.oh-my-zsh/custom/plugins/fzf-dir-navigator-custom.conf
$ <editor command> ~/.oh-my-zsh/custom/plugins/fzf-dir-navigator-custom.conf
```
And then proceed to edit your custom configuration file.

Thanks for the valuable feedback, much appreciated!
